modules/sugar/src/test/ts/browser/ScrollTest.ts

```diff
@@ -127,7 +127,8 @@ UnitTest.asynctest('ScrollTest', (success, failure) => {
     const cX = Math.round(center.left);
     const cY = Math.round(center.top);
 
-    Assert.eq('scroll bar width, got=' + scrollBarWidth, true, scrollBarWidth > 5 && scrollBarWidth < 50 || (platform.os.isMacOS() && scrollBarWidth === 0));
+    const noVisibleScrollbarBrowser = platform.os.isMacOS() || (platform.browser.isFirefox() && platform.os.isLinux()) && (platform.os.isWindows() && platform.os.version.major >= 11);
```

Comment:
```suggestion
    const noVisibleScrollbarBrowser = platform.os.isMacOS() || (platform.browser.isFirefox() && platform.os.isLinux()) || (platform.browser.isFirefox() && platform.os.isWindows() && platform.os.version.major >= 11);
```

Reply:
If this is the same as the other place, should we put it in agar?

Reply:
We should have some form of scroll width feature detection in sand/agar/sugar not sure where it fits I think I've seen code in sugar that measures scrollbar widths.